packages/design-picker/src/utils/available-designs.ts

```diff
@@ -6,6 +6,7 @@ import type { Design, DesignUrlOptions } from '../types';
 import type { AvailableDesigns } from './available-designs-config';
 import type { MShotsOptions } from '@automattic/onboarding';
 
+// Deprecated; used for Gutenboarding (/new flow)
```

Comment:
jsdoc comment would be better here too
```suggestion
/**
 *  @deprecated used for Gutenboarding (/new flow)
 */
```

Reply:
Done!